web/cypress/e2e/builds.cy.ts

```diff
@@ -1031,6 +1031,7 @@ describe('Repository Builds', () => {
 describe('Repository Builds - Create Custom Git Build Triggers', () => {
   beforeEach(() => {
     cy.exec('npm run quay:seed');
+    cy.intercept('GET', '/config', {fixture: 'config.json'}).as('getConfig');
```

Comment:
do we need this?

Reply:
Without this for me the test fails, which makes sense because in the default config builds are disabled and so the builds UI is missing. I have no idea why it worked so far.